iceoryx_dust/container/include/iox/fixed_position_container.hpp

```diff
@@ -317,6 +317,10 @@ class FixedPositionContainer final
         IndexType m_index;
     };
 
+  private:
+    /// @brief Initialize member variables to prevent undefined or erroneous values.
+    void reset_member() noexcept;
```

Comment:
Just one last nitpick. Since you already used `initialize` in the comment could you rename `reset_member` to `init_member` or similar?

Reply:
Well, I decided to remove the function in the last commit. Currently,  a `for` loop is used to initialize `m_status` in ctor.
I think this can be improved in the following PR (refactoring one).